### PR TITLE
address issue #322

### PR DESCRIPTION
**Pull request by giacomomagni**
Address issue #322 .

### CODE REVIEW

**Comment by giacomomagni** [comment]
@felixhekhorn as usual tests of this pieces are quite involved  :( Any idea?

File: src/eko/runner/parts.py [hunk 1]
```diff
@@ -90,7 +90,7 @@ def evolve_configs(eko: EKO) -> dict:
         n3lo_ad_variation=tcard.n3lo_ad_variation,
         matching_order=tcard.matching_order
         if tcard.matching_order is not None
-        else tcard.order,
+        else (tcard.order[0] - 1, tcard.order[1]),
```
Comment (felixhekhorn) [inline src/eko/runner/parts.py:93]:
```suggestion
        else (tcard.order[0] - 1, tcard.order[1] - 1),
```
I know QED matchings are beyond our current scope, let's spend 3 characters now if ever we get there ...

Comment (giacomomagni) [reply src/eko/runner/parts.py:93]: true and also in the legacy...

**Comment by felixhekhorn** [comment]
> @felixhekhorn as usual tests of this pieces are quite involved :( Any idea?

it was not that complicated :upside_down_face: 2cf291d6

File: src/eko/runner/parts.py (hunk 1 above)
Comment (felixhekhorn) [reply src/eko/runner/parts.py]: in the old style `QED` was a boolean, which translated naturally into the new counting - in the new style we still need to translate - do you agree @niclaurenti ? (actually, are we putting this in the fits for the paper to 2?)

Comment (giacomomagni) [reply src/eko/runner/parts.py]: Okay, then it should be fine.
But on the other hand we should add a minus sign here: 
https://github.com/NNPDF/eko/blob/10f0733635c012ce7e1f0fd4cdfc497619292830/src/eko/io/runcards.py#L216

Comment (felixhekhorn) [reply src/eko/runner/parts.py]: why? again in the old style PTO=0 means LO, no?

Comment (giacomomagni) [reply src/eko/runner/parts.py]: no I meant on the the QED

Comment (felixhekhorn) [reply src/eko/runner/parts.py]: okay, yes - just remember to put also a `min` maybe

Comment (niclaurenti) [reply src/eko/runner/parts.py]: > in the old style `QED` was a boolean, which translated naturally into the new counting - in the new style we still need to translate - do you agree @niclaurenti ? (actually, are we putting this in the fits for the paper to 2?)

In the old style `QED` was the order in $alpha$ of the splitting kernels, so if we had QED corrections to matching conditions it should have a `-1` as for the QCD order (In the fit I'm using `QED=2`).

**Review by niclaurenti — APPROVED** [review]

File: src/eko/runner/parts.py [hunk 2]
```diff
@@ -90,7 +90,7 @@ def evolve_configs(eko: EKO) -> dict:
         n3lo_ad_variation=tcard.n3lo_ad_variation,
         matching_order=tcard.matching_order
         if tcard.matching_order is not None
-        else tcard.order,
+        else (tcard.order[0] - 1, tcard.order[1] - 1),
```
[the remark this answers is not in the document]
Comment (giacomomagni) [reply src/eko/runner/parts.py:93]: Okay I agree, let's set it to 0 everywhere, so it's clear what we are doind.

**Comment by giacomomagni** [comment]
@felixhekhorn if you agree with this solution I'll merge asap.

**Review by felixhekhorn — APPROVED** [review]

**Pull request merged**